.circleci/config.yml

```diff
@@ -24,20 +24,20 @@ common: &common
           fi
     - restore_cache:
         keys:
-          - v1-cache-{{ .Environment.CIRCLE_JOB }}-{{ checksum "setup.py" }}-{{ checksum "tox.ini" }}
+          - v2-cache-{{ .Environment.CIRCLE_JOB }}-{{ checksum "setup.py" }}-{{ checksum "tox.ini" }}
     - run:
         name: install dependencies
         command: pip install --user tox
     - run:
         name: run tox
-        command: ~/.local/bin/tox
+        command: ~/.local/bin/tox -- -n 8
```

Comment:
FYI, environment variables work well for this too.  `PYTEST_ADDOPTS="-n 2"`

Also, the ci-environment only has 2 cores so you should probably keep the concurrency down to around 2-4 max unless you know that test running is not CPU bound.

Reply:
Oh, where did you see that?  When I looked at `/proc/cpuinfo` it was claiming there were 8 cores.

Reply:
👍 somewhere in the docs. I can't remember exactly where.  Probably somewhere in the container or parallelization stuff.